index.js

```diff
@@ -252,8 +252,9 @@ function tsLookup({ dependency, filename, tsConfig, tsConfigPath, noTypeDefiniti
     const absoluteBaseUrl = path.join(path.dirname(tsConfigPath), compilerOptions.baseUrl)
     // REF: https://github.com/dividab/tsconfig-paths#creatematchpath
     const tsMatchPath = createMatchPath(absoluteBaseUrl, compilerOptions.paths)
+    const extensions = ['.ts', '.tsx', '.d.ts', '.js', '.jsx', '.json', '.node']
     // REF: https://github.com/dividab/tsconfig-paths#creatematchpath
-    const resolvedTsAliasPath = tsMatchPath(dependency) // Get absolute path by ts path mapping. `undefined` if non-existent
+    const resolvedTsAliasPath = tsMatchPath(dependency, undefined, undefined, extensions) // Get absolute path by ts path mapping. `undefined` if non-existent
```

Comment:
Nit: consider passing an object with keys so that you don't have to supply undefined here. It's easier for the reader and also lets us easily extend to more keys without changing the function signature. The alternative would be "explaining variables" where you create variables just to name the arguments.

Reply:
Nevermind. I see that it's a function that's part of the library you're using.